_includes/releases/v22.2/v22.2.6.md

```diff
@@ -0,0 +1,127 @@
+
+    ~~~sql
+    ASSUME_ROLE={roleA};external_id={idA},{roleB};external_id=<idB>,{roleC}
+    ~~~
+
+    This will use external ID `{idA}` to assume delegate `{roleA}`, then use external ID `{idB}` to assume delegate `{roleB}`, and finally no external ID to assume the final role `{roleC}`. [#96531][#96531]
+- Changefeeds no longer require the `COCKROACH_EXPERIMENTAL_ENABLE_PER_CHANGEFEED_METRICS` environment variable in order to use [`metrics_label`](../v22.2/monitor-and-debug-changefeeds.html#using-changefeed-metrics-labels) option. [#97509][#97509]
+
+<h3 id="v22-2-6-sql-language-changes">SQL language changes</h3>
+
+- Previously, setting a [table's locality](../v22.2/alter-table.html#set-locality) was not allowed if the table contained any hash-sharded indexes. This restriction is now removed. [#96688][#96688]
+- Introduced the `declare_cursor_statement_timeout_enabled` [session variable](../v22.2/set-vars.html), which disables statement timeouts during `FETCH` when using `DECLARE CURSOR`. [#97089][#97089]
+- Added a hard limit of how much data can be flushed to system tables for SQL stats. [#97399][#97399]
+- Added support for expressions of the form `COLLATE "default"`, `COLLATE "C"`, and `COLLATE "POSIX"`. Since the default [collation](../v22.2/collate.html) cannot be changed currently, these expressions are all equivalent. The expressions are evaluated by treating the input as a normal string, and ignoring the collation. This means that comparisons between strings and collated strings that use `"default"`, `"C"`, or `"POSIX"` are now supported. Creating a column with the `"C"` or `"POSIX"` collations is still not supported. [#97415][#97415]
+- Previously, you could use [user-defined function](../v22.2/user-defined-functions.html) usage from tables with `SET DEFAULT` and `SET ON UPDATE` even though they are disallowed from `CREATE TABLE` and `ADD COLUMN`. This patch disallows those two cases from `ALTER TABLE ... ALTER COLUMN`. [#97430][#97430]
+- Increased the default value of `sql.stats.cleanup.rows_to_delete_per_txn` to 10k, to increase efficiency of the cleanup job for SQL statistics. [#97724][#97724]
+
+<h3 id="v22-2-6-operational-changes">Operational changes</h3>
+
+- A [`BACKUP`](../v22.2/backup.html) that encounters too many retryable errors will now fail instead of pausing to allow subsequent backups the chance to succeed. [#96716][#96716]
+
+<h3 id="v22-2-6-db-console-changes">DB Console changes</h3>
+
+- Previously, when the SQL API returned a "max size reached" error, the DB Console was only showing the error, but not the data that was also being returned. Now, the Statement Insights, Transaction Insights, and Schema Insights pages also show the data. [#97470][#97470]
+- Added the following new metrics to track memory usage of prepared statements in sessions: 
```

Comment:
I might just say "Added 22 new metrics to track ... in sessions,." and leave it at that. Up to you.